docs/api/products.rst

```diff
@@ -0,0 +1,32 @@
+========
+Products
+========
+
+This allows list, retrieve, create, update, and delete of Products.
+
+Retrieve
+--------
+
+.. sourcecode:: http
+
+   GET /api/products/ HTTP/1.1
+
+.. sourcecode:: http
+
+   HTTP/1.1 200 OK
+
+    {
+      "count": 150,
+      "next": "https://.../api/products/?page=2",
+      "previous": null,
+      "results": [
+        {
+          "id": "3Lw7OwK",
```

Comment:
@wlonk I imagine we should add a unique `slug` based on `product.name` for use in URLs? Probably also `old_slugs`...